selftests/functional/test_output.py

```diff
@@ -264,16 +264,17 @@ def test_output_compatible_setup_3(self):
         tmpfile = tempfile.mktemp(prefix='avocado_' + __name__)
         tmpfile2 = tempfile.mktemp(prefix='avocado_' + __name__)
         tmpdir = tempfile.mkdtemp(prefix='avocado_' + __name__)
-        tmpfile3 = tempfile.mktemp(dir=tmpdir)
+        tmpfile3 = os.path.join(tmpdir, "result.html")
```

Comment:
The order (tmpfile, tmpfile2, tmpdir, tmpfile3) seemed odd at first, but it makes sense.